content/05.model-complexity.md

```diff
@@ -8,23 +8,23 @@ Third, there is no ambiguity about the labels for the samples in the dataset (i.
 Rare disease datasets violate many of these assumptions.
 There is generally high class imbalance due to small number of samples for specific classes (e.g., only a few patients with a particular rare disease in a health records dataset), the data are often sparse, and there may be abundant label-noise due to incomplete understanding of the disease.
 All of these contribute to low signal to noise ratio in rare disease datasets.
-Applying ML to such data without addressing the aforementioned shortcomings may lead to models that have low reproducibility or are hard to interpret.
+Applying ML to such data without addressing these shortcomings may lead to models that have low reproducibility or are hard to interpret.
 
 Class imbalance in datasets can be addressed using decision tree-based ensemble learning methods (e.g., random forests). [@doi:10.1007/s11634-019-00354-x] (Figure[@fig:3]a)
 Random forests use resampling (with replacement) based techniques to form a consensus about the important predictive features identified by the decision trees (e.g., Box 1c). [@https://doi.org/10.1023/A:1010933404324; @doi:10.1186/1472-6947-13-134]
-Additional approaches like combining random forests with resampling without replacement can generate confidence intervals for the model predictions (e.g., for applications like Box 1d) by iteratively exposing the models to incomplete datasets, mimicking real world cases where most rare disease datasets are incomplete [@doi:10.3390/genes11020226].
+Additional approaches like combining random forests with resampling without replacement can generate confidence intervals for the model predictions (see Box 1d) by mimicking real world cases where most rare disease datasets are incomplete [@doi:10.3390/genes11020226].
```

Comment:
```suggestion
Additional approaches like combining random forests with resampling without replacement can generate confidence intervals for the model predictions (for applications like Box 1d) by mimicking real world cases where most rare disease datasets are incomplete [@doi:10.3390/genes11020226].
```

Reply:
adding this back in since Box 1d refers to a related application, not a direct example.